packages/cbioportal-frontend-commons/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-frontend-commons",
   "description": "cBioPortal Frontend Modules",
-  "version": "0.3.2",
+  "version": "0.3.3-beta.0",
```

Comment:
I needed to manually publish a beta version for the `Signal` test deployment to work. Signal lives outside the cbioportal monorepo, so it needs a published version on npm.

We can keep the beta version in this PR because the lerna publish bot will create and publish the version `0.3.3`. It never publishes pre release versions (alpha, beta, etc).